service/controller/resource/tccp/create.go

```diff
@@ -319,38 +233,6 @@ func (r *Resource) newParamsMain(ctx context.Context, cr infrastructurev1alpha2.
 	return params, nil
 }
 
-func (r *Resource) newParamsMainInstance(ctx context.Context, cr infrastructurev1alpha2.AWSCluster, t time.Time) (*template.ParamsMainInstance, error) {
-	var err error
-	// We need to fetch the ControlPlane CR for once because it needs to be passed to hamaster interface to get the mapopings.
-	var haMapping []hamaster.Mapping
-	{
-		haMapping, err = r.haMaster.Mapping(ctx, &cr)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-	}
-
-	var instance *template.ParamsMainInstance
-	{
-		instance = &template.ParamsMainInstance{
-			Master: template.ParamsMainInstanceMaster{
-				/*
-					This is part of migration code we introduced with the new tccpn stack.
-					We need this to create empty ETCD volume in the first AZ  to do a snapshot
-					and then create volume from that snapshot in the tccpn stack.
-
-					We can take a look at how we could remove it before creating stable release for HA masters.
-				*/
-				AZ: haMapping[0].AZ,
-				EtcdVolume: template.ParamsMainInstanceMasterEtcdVolume{
```

Comment:
we don't need the etcd volume in tccp anymore, so removing the whole template that was left there